packages/radix-ui-themes/src/components/checkbox-card-group.tsx

```diff
@@ -117,5 +117,5 @@ const CheckboxCardGroupItem = React.forwardRef<
 });
 CheckboxCardGroupItem.displayName = 'CheckboxCardGroupItem';
 
-export { CheckboxCardGroupRoot, CheckboxCardGroupItem };
-export type { CheckboxCardGroupRootProps, CheckboxCardGroupItemProps };
+export { CheckboxCardGroupRoot as Root, CheckboxCardGroupItem as Item };
```

Comment:
Turbo nit, but does Prettier coerce us into doing everything on a single line here? I'd prefer multi-line formatting here for consistency and readability

Reply:
prettier ignore?

Reply:
Damn yeah it doesn't, never mind. Not willing to prettier ignore it all. Just thought it's one of those cases when you can do a line break and it'd respect it.

Reply:
This works

<img width="439" alt="CleanShot 2024-03-12 at 14 52 29@2x" src="https://github.com/radix-ui/themes/assets/1539897/7faf6545-02ff-468f-a9be-df386d6c5ba0">

Reply:
Naah not worth it